### PR TITLE
Update markdown documentation

### PR DESCRIPTION
**Pull request by mdahlhausen**
Addresses #1524
Hopefully makes it clearer to people now using openstudio-standards.
We will need to update the documentation again after the refactor that splits things out into modules.

### CODE REVIEW

**Comment by mdahlhausen** [comment]
@asparke2 @lymereJ would appreciate a review and edits of the language in the README on openstudio-standards use-cases.

File: docs/DeveloperInformation.md [hunk 1]
```diff
@@ -81,12 +81,11 @@ This project uses [Rake](http://rake.rubyforge.org/) to run tasks from the termi
...
 
 1. Make a new branch for your changes.
 2. Modify the code on your branch.
-3. Modify the [OpenStudio_Standards Google Spreadsheet](https://drive.google.com/drive/folders/1x7yEU4jnKw-gskLBih8IopStwl0KAMEi?usp=sharing)
- - To get edit access to this spreadsheet, contact <mailto:Marlena.Praprost@nrel.gov>.
+3. If you have data, modify the .json files and run commands to update the database, as appropriate. Historically, openstudio-standards data used a series of google spreadsheets, and is still used for non-90.1- standards. See [OpenStudio_Standards Google Spreadsheet](https://drive.google.com/drive/folders/1x7yEU4jnKw-gskLBih8IopStwl0KAMEi?usp=sharing). Contact <mailto:matthew.dahlhausen@nrel.gov> for edit access.
 4. `bundle exec rake data:update` to download the latest version of the spreadsheet from Google Drive and export the JSON files.
```
Comment (eringold) [inline docs/DeveloperInformation.md:89]: If you provide edit access, developers will still need to obtain a 'client_secret.json' credential file in order for the rake task to work. Maybe this section should include the fact that you can download the spreadsheets and use `bundle exec rake data:update:manual` to update the jsons from the local copies.

File: docs/AddingAStandard.md [hunk 2]
```diff
@@ -1,15 +0,0 @@
-# Adding a Standard
```
[the remark this answers is not in the document]
Comment (mdahlhausen) [reply docs/AddingAStandard.md:1]: It's no longer correct, and I don't think we will support new standards unless we have specific funding to support maintenance for them. That's not to say we won't, just that it's a more involved conversation than just passively suggesting it in a markdown file here.

File: docs/DeveloperInformation.md (hunk 1 above)
Comment (mdahlhausen) [reply docs/DeveloperInformation.md]: Added further detail with [1a3e14a](https://github.com/NREL/openstudio-standards/pull/1576/commits/1a3e14a50c198932756de68a7cc658bbdef47614).

**Pull request merged**